rust/gui-client/src-tauri/src/client/crash_handling.rs

```diff
@@ -144,6 +164,15 @@ impl minidumper::ServerHandler for Handler {
         match result {
             Ok(mut md_bin) => {
                 let _ = md_bin.file.flush();
+                // Copy the timestamped crash file to a well-known filename,
+                // this makes it easier for the smoke test to find it
+                std::fs::copy(
+                    &md_bin.path,
+                    known_dirs::logs()
+                        .expect("Should be able to find logs dir")
+                        .join("last_crash.dmp"),
+                )
+                .ok();
```

Comment:
Unfortunately I can't throw errors here so I just kinda ignore it if writing the dump or copying it fails

Reply:
Windows has symlinks but I don't trust them